views/includes/fixed-footer.pug

```diff
@@ -1,29 +1,15 @@
 div(class='eop')
-  p.discover-heading.sans-serif Discover The Realm of Computer Science Topics!
+  p.discover-heading.sans-serif Codex is based off of the website Medium! Check out our Github links down below
   div(class='tag-grid')
-  button(class='tag-button') Programming
-  button(class='tag-button') Python
-  button(class='tag-button') java
-  button(class='tag-button') C
-  button(class='tag-button') Game programming
-  button(class='tag-button') Data Science
-  button(class='tag-button') Machine Learning
-  button(class='tag-button') Bootcamp
-  button(class='tag-button') arm
+    img(src="./images/codexLogoShort.png", alt="")
```

Comment:
This will technically always work, but I think it's best to update this to `/images/codexLogoShort.png` instead so it's always the intended path regardless of the current route.